controllers/apps/transformer_component_tls.go

```diff
@@ -223,7 +223,7 @@ func (t *componentTLSTransformer) composeTLSVolume(
 	if compDef.Spec.TLS.DefaultMode != nil {
 		volume.VolumeSource.Secret.DefaultMode = ptr.To(*compDef.Spec.TLS.DefaultMode)
 	} else {
-		volume.VolumeSource.Secret.DefaultMode = ptr.To(int32(0600))
+		volume.VolumeSource.Secret.DefaultMode = ptr.To(int32(384))
```

Comment:
![image](https://github.com/user-attachments/assets/58f8c015-f2db-41f2-8790-4bf8d3949dfe)

What's the difference?

Reply:
Fixed. The issue is that the default value set by Kubebuilder is treated as decimal 600 by default.

Reply:
Keep this line unchanged for readability.

Reply:
Done